### DISCUSSION TITLE
Welcome to lazpaint Discussions!

### DISCUSSION THREAD

**Discussion by circular17** [bgrabitmap/lazpaint#233] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lainz** [comment]
Hi, mi name is Leandro, mi nick is lainz. I did the icon for LazPaint =) Report bugs, features and sometimes compile for some targets. Helping with BGRAControls package.

**Reply by circular17** [reply]
Thanks for your help on the project, some things I don't think about you have the idea.

**Comment by c-sanchez** [comment]
Hoho nice idea @circular17 , I can see that you are more active lately with Lazpaint, btw the last version (7.1.6) is really great, well, indeed each new version have great changes and features :D

Honestly what you do with Lazpaint is really cool.

**Reply by circular17** [reply]
Trying to make it better every time :-) You provided a list of requests for possible improvements where I can find what to do next.

**Comment by c-sanchez** [comment]
Photoshop has a custom Title Bar, which in addition to the minimize, maximize and close buttons on the right has the menu bar too.
It may be a small detail but as it is a custom bar the theme blends perfectly with it and saves a little extra space.
Please check how where and how is the **Menu Bar** (**File , Edit, Help, etc**) in both Photoshop and Lazpaint
![image photoshop](https://i.ibb.co/VCDSc96/photoshop-window.png)
![image lazpaint](https://i.ibb.co/NFNMHrC/lazpaint-window.png)

However, the space in lazpaint seems to be better ordered and wider :)
But ok, my point here is about that custom title/menu bar hehe

**Reply by circular17** [reply]
The thing is that it may be complicated to make it work cross platform. Basically this would be based on a borderless window, though from my experience, the only windows that behave more or less the same are the regular ones.

**Reply by lainz** [reply]
It can work only on Windows since the menu bar is on the system on linux and macOS. Maybe we can see on how Visual Studio Code does it since its available on all systems not like photoshop.

**Reply by circular17** [reply]
On MacOS, it is possible to customize a window, though I don't know how much this is integrated into Lazarus.

**Reply by lainz** [reply]
Ok. Anyways we need a custom main menu. But yes on Windows is possible to modify the window too. Like Chrome or Firefox does for example. To put things in the title bar.

**Comment by c-sanchez** [comment]
Yeah, I had thought that this would be quite curious to see in Linux specially, since each Desktop Environment can be very different.
The first one that comes to mind (albeit abandoned) is Ubuntu's Unity, since this when going to full screen combines the menu bar and the minimize / maximize / close buttons of the program with the system bar.
An idea that I think it's really cool from Unity.
![unity desktop](https://i.ibb.co/THwWmgb/unitymenu.gif)

**Reply by circular17** [reply]
Hmm I wonder if that would apply automatically to LazPaint. I suppose so.
